spec/JSON-RPC.md

```diff
@@ -1376,28 +1376,30 @@ Errors: `KVDB Error`, `Invalid Params`
 [Back to **List of methods**](#list-of-methods)
 
 ## chain_getPendingTransactions
-Gets transactions in the current transaction queue.
+Gets transactions whose insertion_timestamps are in the given range from the current transaction queue.
```

Comment:
whose insertion_timestamps are in the given range from -> that have insertion_timestamps within the given range from

Reply:
Fixed. Thank you.